packages/nextjs-mf/src/plugins/container/ModuleFederationPlugin.ts

```diff
@@ -9,8 +9,15 @@ import { ModuleFederationPluginOptions } from './types';
 import SharePlugin from '@module-federation/enhanced/src/lib/sharing/SharePlugin';
 import ContainerPlugin from '@module-federation/enhanced/src/lib/container/ContainerPlugin';
 import ContainerReferencePlugin from '@module-federation/enhanced/src/lib/container/ContainerReferencePlugin';
-const createSchemaValidation = require('webpack/lib/util/create-schema-validation');
-const isValidExternalsType = require('webpack/schemas/plugins/container/ExternalsType.check.js');
+import { normalizeWebpackPath } from '@module-federation/sdk/normalize-webpack-path';
+const createSchemaValidation = require(
+  normalizeWebpackPath('webpack/lib/util/create-schema-validation'),
```

Comment:
Next uses fork of MFP upper plugin to workaround some issues caused by lack of ability to control shared fully. S